subprojects/docs/src/docs/userguide/getting_started.adoc

```diff
@@ -21,7 +21,7 @@ Everyone has to start somewhere and if you're new to Gradle, this is where to be
 
 In order to use Gradle effectively, you need to know what it is and understand some of its fundamental concepts. So before you start using Gradle in earnest, we highly recommend you read <<what_is_gradle#what_is_gradle,What is Gradle?>>.
 
-Even if you're experienced with using Gradle, we suggest you read the section <<what_is_gradle#five_things,5 things you need to know about Gradle>> as it clears up some common misconceptions.
+Even if you're an experienced Gradle user, we suggest you read "<<what_is_gradle#what_is_gradle,What is Gradle?>>" as it clears up some common misconceptions.
```

Comment:
Doesn't this read a little weird now?

The first paragraph says "Read What is Gradle?" and the second paragraph says "No, really, Read What is Gradle?"